pyv/inference.py

```diff
@@ -0,0 +1,154 @@
+import ast
+
+from py2many.inference import get_inferred_type, InferTypesTransformer
+from py2many.analysis import get_id
+
+
+V_TYPE_MAP = {
+    "int": "int",
```

Comment:
Should be BigInt to be correct IMHO.

Reply:
While it'd be more correct, the performance impact would be a lot. Not a lot people go to values higher than i64.

Reply:
Then it should at least be `i64`, right? V's default `int` is `i32` if I'm not mistaken.

Reply:
Probably a V bug but you can't `i64(1) in [1, 2, 3]` currently which breaks a test case. I will leave it as it is for now.

Reply:
We chose to go with `int` or `i32` in other backends.

The reasoning is that we can't obviously provide 100% compatibility with python's arbitrary precision `int` without making serious compromises on efficiency and still deliver on the promise of being able to write secure integer code without overflows/underflow errors using fixed width ints.

I like to think about the subset of python supported by py2many as a new statically typed programming language with the semantics of the language we're transpiling to, but with the syntax of python (so we can leverage the ecosystem to the extent that we can).